public/scripts/power-user.js

```diff
@@ -1602,18 +1603,8 @@ async function loadPowerUserSettings(settings, data) {
     loadCharListState();
 }
 
-async function loadCharListState() {
```

Comment:
the code suggests it was trying to alter the class before the thing existed in the DOM..or it was trying to apply a class based on the value of a power_user property that hadn't loaded yet. maybe a relic of a bygone era.